notify/slack/slack.go

```diff
@@ -92,13 +93,22 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 		tmplText = notify.TmplText(n.tmpl, data, &err)
 	)
 	var markdownIn []string
+	key, err := notify.ExtractGroupKey(ctx)
+
+	if err != nil {
+		return false, err
+	}
```

Comment:
i would move this block inside the `if truncated { ... }` block.

Reply:
Done